csrc/scheduler/normalization_inner_outer.h

```diff
@@ -26,6 +26,9 @@ class HeuristicSummary;
 
 class InnerOuterPersistentKernelScheduler : public SchedulerEntry {
  public:
+  constexpr static int64_t threads_per_block_min = 128l;
+  constexpr static int64_t threads_per_block_max = 256l;
```

Comment:
where does this 256 coming from?

Reply:
It comes from the previous implementation. In this PR I just gave these magic numbers some names without changing them.
why 256 ? It is because this scheudler has very high register pressure due to extra registers to store intermediate outer reduction results. So prefer to allow 255 registers per thread and then the max threads per block is 256. (64K/256=256). May be revised in the heuristic revision PR.